src/main/g8/src/main/resources/application.conf

```diff
@@ -6,16 +6,24 @@ akka {
 
 api {
   host = "0.0.0.0"
+  port = ${?API_HOST}
   port = 8080
+  port = ${?API_PORT}
 }
 
 db {
   db_name = "items"
+  db_name = ${?DB_NAME}
   host = "localhost"
+  host = ${?DB_HOST}
   port = 5432
-  url = "jdbc:postgresql://"\${db.host}":"\${db.port}"/"\${db.db_name}
+  port = ${?DB_PORT}
+  
+  url = "jdbc:postgresql://"${db.host}":"${db.port}"/"${db.db_name}
```

Comment:
The backslashes before `$` are needed for escaping. Without it the template fails to generate a project.
The tests should pick that up but somethings wrong with them at the moment.

Reply:
Thanks for the explanation (first time contributing to a g8!),
I've just escaped all those`$`.